lib/faraday_middleware/response/caching.rb

```diff
@@ -48,7 +51,11 @@ def call(env)
             response = @app.call(env)
 
             if CACHEABLE_STATUS_CODES.include?(response.status)
-              cache.write(key, response)
+              if @options[:write_options]
```

Comment:
I've added this `if` statement with backwards compatibility in mind - if people are using `ActiveSupport::Cache::Store`-confirming objects, this will work fine, but people may not be, especially in tests, so we shouldn't start passing options if the user hasn't set some.